### PR TITLE
8144030: [macosx] test java/awt/Frame/ShapeNotSetSometimes/ShapeNotSetSometimes.java fails (again)

### PR DESCRIPTION
**Pull request by azvegint**
This test has two undecorated frames: 

1. background, 300x300
2. shaped foreground, 200x200

After displaying this frames it tries to check color outside of shape but within frame 2 bounds.

Unfortunately, on Linux and macOS there is a shadow around the shaped frame:
image
(changed color to green to make it more noticeable).

One way to fix the test would be to add tolerance to color check.
But with old check points blue color may vary in 222-255 range, which is a bit too much.

Instead of this I made this shaped window  the size of background window. 
This allows us to move check points away from shape border and its shadow, but still be able to check whether shape is applied or not.

Testing is green on all platforms.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8144030](https://bugs.openjdk.java.net/browse/JDK-8144030): [macosx] test java/awt/Frame/ShapeNotSetSometimes/ShapeNotSetSometimes.java fails (again)

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8233/head:pull/8233` \
`$ git checkout pull/8233`

Update a local copy of the PR: \
`$ git checkout pull/8233` \
`$ git pull https://git.openjdk.java.net/jdk pull/8233/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8233`

View PR using the GUI difftool: \
`$ git pr show -t 8233`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8233.diff">https://git.openjdk.java.net/jdk/pull/8233.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/awt/Frame/ShapeNotSetSometimes/ShapeNotSetSometimes.java [hunk 1]
```diff
@@ -124,48 +125,49 @@ public static void main(String[] args) throws Exception {
...
-                        "expected background color " + BACKGROUND_COLOR);
-                throw new RuntimeException("Test failed. The shape has not been applied.");
+            colorCheck(innerPoint.x, innerPoint.y, SHAPE_COLOR);
+
+            for (Point point : pointsOutsideToCheck) {
+                colorCheck(
+                        point.x,
+                        point.y,
+                        BACKGROUND_COLOR
+                );
```
Comment (azvegint) [inline test/jdk/java/awt/Frame/ShapeNotSetSometimes/ShapeNotSetSometimes.java:144]: Dunno why the old version was not checking inner points, but I've added one just to check if the shaped frame is above background one.

Comment (mrserb) [reply test/jdk/java/awt/Frame/ShapeNotSetSometimes/ShapeNotSetSometimes.java:144]: Don't we need to check the places where the shadow is rendered that it is not the shape color(to prove that the shape is not simple rectangular)?

Comment (azvegint) [reply test/jdk/java/awt/Frame/ShapeNotSetSometimes/ShapeNotSetSometimes.java]: Sure, extra check added.

**Review by mrserb — APPROVED** [review]

**Pull request closed without merging**